fsd_config/form_jsons/dpi_r2/your-skills-and-experience-dpi.json

```diff
@@ -64,8 +64,8 @@
                         "maxWords": "250"
                     },
                     "type": "FreeTextField",
-                    "title": "Tell us about how you have worked on similar previous projects",
-                    "hint": "<div class=\"govuk-hint\">Tell us about:\n<ul class=\"govuk-list govuk-list--bullet govuk-hint\">\n  <li>the scale of the project</li>\n  <li>the value of the project</li>\n  <li>where the project was based</li>\n</ul>\n</div>"
+                    "title": "Tell us about how your organisation has worked on similar previous projects",
```

Comment:
@adamdavies1 The title should be "Tell us about how your organisation has worked on similar previous projects" as per this Mural:
https://app.mural.co/t/mhclg2837/m/mhclg2837/1696261529179/b2f591eb34e686a40eac085e1d3980dbf4cc0198?fromVisitorModal=true&sender=uf71d8c9727ac72dd288d0133